editor/animation_bezier_editor.cpp

```diff
@@ -1231,6 +1231,17 @@ void AnimationBezierTrackEdit::gui_input(const Ref<InputEvent> &p_event) {
 			EditorUndoRedoManager *undo_redo = EditorUndoRedoManager::get_singleton();
 			undo_redo->create_action(TTR("Add Bezier Point"));
 			undo_redo->add_do_method(animation.ptr(), "bezier_track_insert_key", selected_track, time, new_point[0], Vector2(new_point[1], new_point[2]), Vector2(new_point[3], new_point[4]));
+			int k_idx = animation->track_find_key(selected_track, time) + 1;
+			switch (int(EDITOR_GET("editors/animation/default_bezier_key_behavior"))) {
```

Comment:
Must be cached in `AnimationBezierTrackEdit::_notification()`.

Reply:
done. maybe you can check if I did it the right way.